src/e2e-test/features/spanner/sink/GCSToSpanner.feature

```diff
@@ -38,6 +75,7 @@ Feature: Spanner Sink - Verification of GCS to Spanner data transfer
     Then Connect source as "GCS" and sink as "Spanner" to establish connection
     Then Open GCS source properties
     Then Enter the GCS source mandatory properties
+    Then Click on the Get Schema button
```

Comment:
Again same comment ->

Do not change existing scenario for @SPANNER_SINK_NEWDB_TEST. Keep this scenario as it is.
Scenario: To verify data is getting transferred successfully from GCS to Spanner new database